spikesorters/yass/example.py

```diff
@@ -0,0 +1,40 @@
+from spikeextractors import YassSortingExtractor
+from yass import YassSorter
+
+
+# INITIALIZE A TOY DATASET
+if __name__ == "__main__":
+    
+    import spikeextractors as se
+    rec, sort = se.example_datasets.toy_example(duration=300)
+    
+    # INITIALIZE YASS SORTER OBJECT
+    output_folder = '/media/cat/4TBSSD/spikeinterface/'
+    sorter = YassSorter()
```

Comment:
@catubc  You need to pass the argument: `recording=rec, `output_folder=output_folder`